Makefile

```diff
@@ -39,6 +39,10 @@ build-multiplatform:
 	CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 $(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) \
 	     -o mender-cli.darwin.amd64
 
+build-coverage:
+	CGO_ENABLED=0 $(GO) test -c -o mender-cli-test \
+		-coverpkg $(shell $(GO) list ./... | grep -v vendor | grep -v mocks | grep -v test | tr  '\n' ,)
```

Comment:
Nit:
`go list ./...` never lists the vendor dir

Reply:
Good to know. It is probably wrong everywhere ([inventory](https://github.com/mendersoftware/inventory/blob/master/Dockerfile.acceptance-testing#L4-L5) as an example)